File: fmarket/src/main/java/ro/fmarket/admin/demand/DemandDetailsDTO.java
package ro.fmarket.admin.demand;

import java.util.ArrayList;
import java.util.List;

import org.joda.time.DateTime;

import lombok.Data;
import ro.fmarket.model.account.consts.AccountStatus;
import ro.fmarket.model.domain.demand.DemandDomain;

@Data
public class DemandDetailsDTO {

	private int id;
	private String title;
	private String message;
	private List<String> cities = new ArrayList<>();
	private boolean allCities;
	private String phone;
	private DateTime creationDate;
	private DateTime activationDate;
	private DemandDomain domain;
	private String name;
	
	/* Account */
	
	private int accountId;
	private String accountEmail;
	private AccountStatus accountStatus;
	private int demandsCount;
	
	
}

File: fmarket/src/main/webapp/app/models/demandDetailsDTO.ts
import {AccountStatus} from "./accountStatus";
/**
 * Created by NicolaeB on 4/26/2016.
 */
export class DemandDetailsDTO{
    public id:number;

    public title:string;

    public message:string;

    public cities:Array<string> = new Array<string>();

    public allCities:boolean;

    public phoneContact;boolean ;

    public creationDate:Date;

    public activationDate:Date;

    /* Account */

    public accountId:number;

    public accountEmail:string;

    public accountStatus:AccountStatus;

    public demandsCount:number;

    public isInEditMode:boolean=false;
}